README.rst

```diff
@@ -94,6 +94,65 @@ Example
     except KeyboardInterrupt:
         loop.run_until_complete(handler.finish_connections())
 
+Same example with asynchronous context manager interface (python3.5+)
+
+.. code:: python
+
+    import asyncio
+    from aiohttp.web import Application, Response
+    from aiohttp_sse import EventSourceResponse
+
+
+    async def hello(request):
+        async with EventSourceResponse(request=request) as resp:
+            for i in range(0, 100):
+                print('foo')
+                await asyncio.sleep(1, loop=loop)
+                resp.send('foo {}'.format(i))
+        return resp
+
+
+    async def index(request):
+        d = b"""
+            <html>
+            <head>
+                <script type="text/javascript"
+                    src="http://code.jquery.com/jquery.min.js"></script>
+                <script type="text/javascript">
+                var evtSource = new EventSource("/hello");
+                evtSource.onmessage = function(e) {
+                 $('#response').html(e.data);
+                }
+
+                </script>
+            </head>
+            <body>
+                <h1>Response from server:</h1>
+                <div id="response"></div>
+            </body>
+        </html>
+        """
+        return Response(body=d)
+
+
+    async def init(loop):
+        app = Application(loop=loop)
+        app.router.add_route('GET', '/hello', hello)
+        app.router.add_route('GET', '/index', index)
+
+        handler = app.make_handler()
```

Comment:
`aiohttp` now has helpers like `web.run_app(app)` see example here: http://aiohttp.readthedocs.io/en/stable/tutorial.html#application but probably better to do this in separate PR